sources/api/migration/migrations/archived/v1.2.0/cfsignal/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "cfsignal"
```

Comment:
This migration crate should go into a separate directory.
We're currently on v1.6.0. We should rebase onto the latest develop so we can see the new migrations.
Since this feature would be introduced in the next minor version, the path of this migration crate should be:
`sources/api/migration/migrations/v1.7.0/cfsignal/`

I also recommend renaming this crate to `add-cfsignal` to not get mixed up with the actual `cfsignal` crate.

This migration also needs to be listed in `Release.toml` and `sources/Cargo.toml` along with the other migrations. Here's an example of what the two changes look like for any singular migration: https://github.com/bottlerocket-os/bottlerocket/pull/1898/commits/082e231c70f2d36a64b3edf6be25e47395e87da6